### PR TITLE
Tidy up ALF X1.1

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

This board appears to support a couple of layouts, but the only one currently specified is a TKL with a split right shift. It also seems to support ISO enter, but not the split left shift that normally comes with an ISO layout - should I add this, and if so, what should it be named? `tkl_ansi_isoenter`?

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Do you have this board and can verify the layout? There is only one layout because that is all that was contained within the jsons that Alf sent me. I didn't want to proceed with adding any more until I had confirmation, and it never came.

**Comment by fauxpark** [comment]
I don't have this board, no, but the supported layouts seem to be shown here (forgot to link it, sorry): https://drop.com/buy/alf-studio-x1-1-custom-mechanical-keyboard-kit

**Comment by mechmerlin** [comment]
I would be weary of merging this change without the testing to verify the new layouts. 

The QMK Configurator is meant to be an easy to use interface for those unaccustomed to creating dev environments or coding. If any of these layouts are incorrect, it would be hard for a user to get things remedied. On top of that due to some controversy, Alf has been MIA and users will not be able to reach out to them/him for support. I would hold off on this change until we can verify it or if you have a very high confidence that all configurations have the correct switches selected.

**Comment by fauxpark** [comment]
Fair enough, looking closer I can't be sure that the right GUI in the 7u layouts isn't sharing the matrix position of the menu key... here's a tidy up that doesn't touch the layouts instead.

File: keyboards/alf/x11/keymaps/default/keymap.c [hunk 1]
```diff
@@ -15,64 +15,23 @@
...
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),                KC_UP,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_RGUI, KC_RCTL,     KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT(
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, RESET,
+
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+        BL_TOGG, BL_DEC,  BL_INC,  BL_STEP, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     KC_MUTE, KC_MPLY, KC_MSTP,
+        RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______, _______, _______,
```
Comment (noroadsleft) [inline keyboards/alf/x11/keymaps/default/keymap.c]: You have an extra keycode here that's preventing this keymap from compiling.

```suggestion
        RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______,          _______,
```

Comment (fauxpark) [reply keyboards/alf/x11/keymaps/default/keymap.c]: Yeah, I noticed 😄

**Pull request merged**